src/adaptation/icap/Xaction.cc

```diff
@@ -14,8 +14,8 @@
 #include "adaptation/icap/Launcher.h"
 #include "adaptation/icap/Xaction.h"
 #include "base/AsyncCallbacks.h"
+#include "base/IoManip.h"
```

Comment:
How did this creep in? It doesn't seem related to the changes in the file

Reply:
It looks unrelated, but it is required to provide our custom std:: optional "printing" operator that used to be in Optional.h.

Reply:
I had figured that out, but hadn't seen that used in the changed code

Reply:
IIRC, it is used for printing the following async callback (parameter). The code in question creates a call(back) object in icapLookupDnsResults() and, hence, has to debugs() it.

```C++
Adaptation::Icap::Xaction::dnsLookupDone(std::optional<Ip::Address> addr)
```